build-info-extractor-gradle/src/main/groovy/org/jfrog/gradle/plugin/artifactory/extractor/GradleDeployDetails.java

```diff
@@ -48,14 +48,17 @@ public Project getProject() {
     public PublishArtifactInfo getPublishArtifact() {
         return publishArtifact;
     }
-    
+
     public int compareTo(GradleDeployDetails that) {
         if (this.publishArtifact == null) {
             return -1;
         }
         if (that.publishArtifact == null) {
             return 1;
         }
+        if (this.deployDetails.compareTo(that.deployDetails) == 0) {
```

Comment:
`compareTo` triggers a big string comparison. Can you please save the result in a variable and reuse it in the last line?

Reply:
Done